test/git.yml

```diff
@@ -179,3 +179,30 @@
     - fail:
         msg: "File bar.html does not exist"
       when: st.stat.exists is not defined or st.stat.exists == False or st.stat.isdir == True
+
+- name: When deploying using Git (remote path for identity key)
+  hosts: all
+  vars:
+    ansistrano_deploy_via: "git"
+    ansistrano_git_repo: git@github.com:fjros/ansistrano-git-test.git
```

Comment:
Please change this to git@github.com:ansistrano/git-test.git which will be the repo with your key